Products/CMFPlone/controlpanel/browser/usergroups_groupdetails.py

```diff
@@ -2,13 +2,28 @@
 from plone.base import PloneMessageFactory as _
 from plone.protect import CheckAuthenticator
 from Products.CMFCore.utils import getToolByName
-from Products.CMFPlone.controlpanel.browser.usergroups import (
-    UsersGroupsControlPanelView,
-)
+from Products.CMFPlone.controlpanel.browser.usergroups import UsersGroupsControlPanelView
 from Products.statusmessages.interfaces import IStatusMessage
-
+import re
 
 class GroupDetailsControlPanel(UsersGroupsControlPanelView):
+    def validate_email(self, email):
+        """Validate email format using regex pattern."""
+        pattern = r'^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$'
+        return re.match(pattern, email) is not None
```

Comment:
Thanks a lot for your contribution :)

There is for sure some email validator already in our code base. 
Please try to find and reuse the existing code!